src/scss/elements/_anchor.scss

```
@@ -0,0 +1,11 @@
+.anchor-btn{
+    background-color: #337099;
```

We should use a variable here which we already have for this colour in the _variables.scss file
```suggestion
    background-color: $astral;
```